museek/antenna_sanity/from_log.py

```diff
@@ -24,6 +24,6 @@ def straggler_list(self) -> list[str]:
                 for antenna_name in addition:
                     straggler_list.append(antenna_name[1:-1])
 
-                unique_straggler_list = sorted(set(straggler_list))
+        unique_straggler_list = sorted(set(straggler_list))
```

Comment:
please check unittests before merging

Reply:
I update the unittests for from_log and time_analysis.  from_log is ok.  time_analysis is ok after a small correction.